unabridged-works-volume-3/original/08-holy-truth-defended.adoc

```diff
@@ -1683,14 +1799,18 @@ Yea, the Spirit of Christ is so necessary and inseparable from him that is Chris
 that the apostle expressly affirms, that "`if any man have not the Spirit of Christ,
 he is none of his.`" Rom. 8:9.
 
-Fourthly, They tasted of the good Word of God.
+[.numbered]
+_Fourthly,_ They tasted of the good Word of God.
 
 Of that Word from which the gift comes;
 of that Word which was in the beginning of the world,
 which is ingrafted into the hearts of those that truly believe;
 which Word is able to save the soul.
 
-Fifthly, They tasted of the powers of the world to come.
+[.numbered]
+_Fifthly,_ They tasted of the powers of the world to come.
+
+====
```

Comment:
I think that the next paragraph is part of this _fifth_ statement of his, let's move the closing `====` down to line 1814 👍